src/type_conversions/rpc/trace.rs

```diff
@@ -366,7 +426,11 @@ impl ToReth<TransactionTrace> for EthersTransactionTrace {
             trace_address: self.trace_address,
             subtraces: self.subtraces,
             action: self.action.into_reth(),
-            result: self.result.into_reth(),
+            result: match self.result.into_reth() {
```

Comment:
`TransactionTrace::result` is `Option<TraceOutput>` so if we encounter EthersRes::None, just return `Option::None`, because there is no equivalent `TraceOutput::None`.